mongo/change_stream.go

```diff
@@ -325,12 +325,6 @@ AggregateExecuteLoop:
 			}
 			defer conn.Close()
 
-			// If wire version is now < 6, do not retry.
-			cs.wireVersion = conn.Description().WireVersion
```

Comment:
Do we need to update the `wireVersion`?

Reply:
Ah yes, I think we should. Nice catch, thank you!